### ISSUE TITLE
Card: Title violates accessibility

### ISSUE THREAD

**Issue by miedrl** [primefaces/primefaces#9527]
### Describe the bug

For fulfilling [WCAG success criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Techniques/html/H42)  "HTML heading markup has to be used to provide semantic code for headings" so headlines should contain a h1-h6 tag. The title of p:card does not fulfil this criterion as it is a <div> tag.

![grafik](https://user-images.githubusercontent.com/77431692/208651362-9e2bf5d7-6836-47d5-9431-d22f386fb844.png)

### Expected behavior

The title of p:card should not be a `<div>` but an h1-h6 tag.

### PrimeFaces edition

Elite

### PrimeFaces version

11.0.0

### Theme

Diamond

### JSF implementation

MyFaces

### JSF version

x

### Java version

x

**Comment by melloware** [comment]
@jepsar if I change to h6 which is what it should be the styles are still respected in the PF theme and overrides default H6 so I think there are no theme changes here.

**Comment by jepsar** [comment]
Let me try in the showcase. There might be subtile changes.

**Comment by melloware** [comment]
ok cool

**Comment by jepsar** [comment]
There is a really small change in height. But more problematic is changing to h6. Basically you cannot use a fixed header level, or else you might violate other accessibility rules. See https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks

**Comment by melloware** [comment]
@miedrl what are your thoughts @jepsar is saying we can't use a fixed H in the rendered.

**Comment by miedrl** [comment]
Okay, changing to a fixed header level seems to be problematic. Maybe one has to add the h1-h6 to the card depending on the general page structure.

**Comment by jepsar** [comment]
Can't you use the title facet to do so?

**Comment by melloware** [comment]
yeah you could just use the facet...

```xml
<f:facet name="title">
        <h2>My Header</h2>
    </f:facet>
```

**Comment by melloware** [comment]
Closing as we can't fix this and a workaround has been provided.

**Comment by miedrl** [comment]
That's exactly what we have done as a workaround 👍

**Comment by jepsar** [comment]
By the way, we could add a heading component to PFE, where you could provide the level as a parameter. I think I've created something similar in some project here.

For example:

```xhtml
<pe:heading level="#{bean.currentLevel + 1}">Hello world</pe:heading>
```

**Comment by melloware** [comment]
I would be OK with that!

**Comment by jepsar** [comment]
See also https://forum.primefaces.org/viewtopic.php?f=136&t=78295
